.npmrc

```diff
@@ -0,0 +1,2 @@
+PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD=1
```

Comment:
we don't use playwright, but backstop has a dependency on it and when they added that all our build started downloading binaries for all playwright browsers unnecessarily - this disable that behaviour